src/popcorn-list-movies/PopcornListMovies.js

```diff
@@ -66,6 +66,14 @@ export class PopcornListMovies extends LocalizeMixin(
     }
   }
 
+  isMovieImageValid(movie) {
+    if(movie.poster === "N/A") {
+      return "https://ih1.redbubble.net/image.512138487.5983/fposter,small,wall_texture,product,750x1000.u3.jpg"
+    } else {

```

Comment:
You will always return something in this method, so in this case you dont need the "else" statement, with the "if" is enough.